modules/virt-runbook-virtualmachinecrcerrors.adoc

```diff
@@ -27,41 +27,45 @@ severe performance degradation.
 [id="diagnosis-virtualmachinecrcerrors"]
 == Diagnosis
 
-. Get the volume name from a virtual machine:
+. Navigate to *Observe* -> *Metrics* in the web console.
+. Obtain a list of VirtualMachines with incorrectly configured storage classes
+by running the following PromQL query:
 +
-[source,terminal]
+[source,text]
 ----
-$ oc get vm <vm_name> -o jsonpath='{.spec.template.spec.volumes}'
+kubevirt_ssp_vm_rbd_volume{rxbounce_enabled="false", volume_mode="Block"} == 1
 ----
-
-. Get the storage class name from the volume:
 +
-[source,terminal]
+The output displays a list of VirtualMachines that use a storage
+class without `rxbounce_enabled`.
++
+.Example output
++
+[source,text]
 ----
-$ oc get pvc <volume> -o jsonpath='{.spec.storageClassName}'
+kubevirt_ssp_vm_rbd_volume{name="testvmi-gwgdqp22k7", namespace="test_ns", pv_name="testvmi-gwgdqp22k7", rxbounce_enabled="false", volume_mode="Block"} 1
 ----
 
-. Get the storage class configuration:
+. Obtain the storage class name by running the following command:
 +
 [source,terminal]
 ----
-$ oc get sc <storage_class> -o yaml
+$ oc get pv ${PV_NAME} -o=jsonpath='{.spec.storageClassName}'
```

Comment:
Would `PV_NAME` have been set already by this point? Or is it intended to be the value of `pv_name` from the previous step's output?